### PR TITLE
Update commands to reinit nodes

### PR DESCRIPTION
**Pull request by jcstein**
## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

File: docs/nodes/celestia-node.mdx [hunk 1]
```diff
@@ -444,17 +478,31 @@ Error: nodebuilder/share: interval must be positive; nodebuilder/core: invalid I
 Error: nodebuilder/share: interval must be positive
 ```
 
-You can re-initialize your node with the following command:
+You can re-initialize your node with the following commands:
```
Comment (renaynay) [inline docs/nodes/celestia-node.mdx:481]: wait is this only for the config related issue? Check the heading

**Review by renaynay — CHANGES_REQUESTED** [review]

File: docs/nodes/celestia-node.mdx [hunk 2]
```diff
@@ -444,17 +478,31 @@ Error: nodebuilder/share: interval must be positive; nodebuilder/core: invalid I
...
+
+  ```sh
+  celestia light start --p2p.network blockspacerace
+  ```
+
+Here's an example for blockspacerace:
 
 ```sh
-rm -rf ~./celestia-light-blockspacerace && celestia light init --p2p.network blockspacerace
+rm -rf ~./celestia-light-blockspacerace
```
[the remark this answers is not in the document]
Comment (jcstein) [reply docs/nodes/celestia-node.mdx:504]: good catch

File: docs/nodes/celestia-node.mdx [hunk 3]
```diff
@@ -432,7 +432,41 @@ celestia light start --p2p.network blockspacerace --core.ip <address> --gateway
...
-### Errors
+</TabItem>
+<TabItem value="mocha" label="Mocha">
+
+```sh
+celestia light init --p2p.network mocha 
+celestia light start --p2p.network mocha --core.ip <address> --gateway --gateway.addr <ip-address> --gateway.port <port>
+```
+
+> NOTE: It is advised before switching networks to reinitialize
```
[the remark this answers is not in the document]
Comment (jcstein) [reply docs/nodes/celestia-node.mdx:443]: the steps to remove `<nodestore>/data` is above this start command, as well as the re-init command

File: docs/nodes/celestia-node.mdx (hunk 1 above)
Comment (jcstein) [reply docs/nodes/celestia-node.mdx:481]: what do you mean? heading is "Clearing data store"

Comment (jcstein) [reply docs/nodes/celestia-node.mdx:481]: https://docs.celestia.org/pr-preview/pr-551/nodes/celestia-node/#clearing-data-store

File: docs/nodes/celestia-node.mdx [hunk 4]
```diff
@@ -444,138 +443,98 @@ Error: nodebuilder/share: interval must be positive; nodebuilder/core: invalid I
...
-
-If you an encounter an error, it is likely that an old config file is present:
-
-```sh
-Error: nodebuilder/share: interval must be positive; nodebuilder/core: invalid IP addr given:
+2. Re-initialize the node:
 
-# or
+  ```sh
+  celestia light init --p2p.network <network>
```
[the remark this answers is not in the document]
Comment (jcstein) [reply docs/nodes/celestia-node.mdx:461]: this is an example

**Review by renaynay — APPROVED** [review]

**Pull request merged**